eslint.config

```diff
@@ -1,9 +1,21 @@
 {
-  "parserOptions": { "ecmaVersion": 6 },
-  "extends": "airbnb-base",
-    rules: {
-    'no-shadow': 0,
-    'no-param-reassign': 0,
-    'eol-last': 0,
+  "parserOptions": {"ecmaVersion": 9},
+  "env": {
+    "browser": true,
+  },
+  "extends": ["airbnb-base", "plugin:react/recommended"],
+  "settings": {
+    "react": {
+      "pragma": "el"
+    }
+  },
+  rules: {
+    "eol-last": 0,
+    "no-shadow": 0,
+    "react/jsx-key": "off",
+    "no-param-reassign": 0,
+    "react/prop-types": "off",
+    "arrow-parens": "as-needed",
+    "react/no-string-refs": "off"
   }
-}
+}
```

Comment:
Reason as the previous

Reply:
I changed The eslint.json because the default one from Microverse for Javascript doesn't take into consideration some React behavior I added in my code.